### PR TITLE
TELCODOCS-470 -  enterprise-4.11 release note for SNO worker nodes feature

### PR DESCRIPTION
**Pull request by aireilly**
RN for SNO worker nodes.

* Merge to enterprise-4.11

Issue:
https://issues.redhat.com/browse/TELCODOCS-470

Link to docs preview:
http://file.emea.redhat.com/aireilly/telcodocs-470-sno-workers-rn/release_notes/ocp-4-11-release-notes.html#ocp-4-11-nodes-worker-nodes

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -589,6 +589,13 @@ The Self Node Remediation Operator provides the following enhancements:
 
 For more information, see xref:../nodes/nodes/eco-self-node-remediation-operator.adoc#self-node-remediation-operator-remediate-nodes[Remediating nodes with the Self Node Remediation Operator].
 
+[id="ocp-4-11-nodes-worker-nodes"]
+==== Worker nodes for {sno} clusters
+
+You can now add worker nodes to {sno} clusters. This is useful for deployments in constrained environments or at the network edge when you need to add additional capacity to your cluster.
```
Comment (Amrita42) [inline release_notes/ocp-4-11-release-notes.adoc:595]: is it **constrained or resource-constrained** environment?  When I looked at the OCP docs, I could see instances of resource-constrained.

Comment (aireilly) [reply release_notes/ocp-4-11-release-notes.adoc]: Updated.

**Comment by Amrita42** [comment]
lgtm

**Pull request merged**